### ISSUE TITLE
support /names

### ISSUE THREAD

**Issue by fauno** [haavard/tox-weechat#12]
hi! thanks for this plugin!

it may be a corner case, but since i hide the nicklist i use /names to print the nicklist when i need it, would you support /names to mimic irc's behaviour?

also /query to open a private message buffer :)

**Comment by haavard** [comment]
Thanks for the feedback! The commits above add the `/names` and `/query` commands. Could you close the issue if they work as you'd expect?

**Comment by fauno** [comment]
thanks!  i'll test it when someone is online :P

though i would expect it to work on the tox profile buffer also, since
it has a nicklist too, what do you think?

**Comment by haavard** [comment]
I agree that `/names` should work in the profile buffer as well. Will take a look at it.

**Comment by fauno** [comment]
just tested `/query`, since its an alias to `/msg` it doesn't do anything if you don't put a message, so `/query person` doesn't work, but `/query person hi!` does :)
